NEW add configuration argument to wizard command (#200)
GoogleChrome/lighthouse-ci: docs/configuration.md

};
```

### Custom headers for running the wizard

> If you're running the `lhci server` behind an reverse proxy or any other component that requires some `extra headers` you can configure them in the wizard section `extraHeaders`.

Reviewer: ```suggestion
If you're running the `lhci server` behind a reverse proxy or any other component that requires some extra headers you can configure them in the wizard section `extraHeaders`.
```

```json
{
  "ci": {
    "wizzard": {
      "extraHeaders": "{\"Authorization\": \"Basic content\"}"
    }
  }
}
```

> > **_NOTE:_** The `wizard` options will be overriten by the `upload` options, to be sure that the wizard options will be used, create a seperate config file

Reviewer: ```suggestion
> **_NOTE:_** The `wizard` options will be overwritten by the `upload` options, to be sure that the wizard options will be used, create a separate config file.
```

### Default lhci server location for running the wizard

> If you're running the `lhci wizard` multiple times, you can configure a default `serverBaseUrl` that you have to type them in at each `lhci wizard` run.

Reviewer: ```suggestion
If you're running the `lhci wizard` multiple times, you can configure a default `serverBaseUrl` to avoid typing it in at each `lhci wizard` run.
```

```json
{
  "ci": {
    "wizzard": {
      "serverBaseUrl": "https://localhost:3000/",
    }
  }
}
```

> > **_NOTE:_** The `wizard` options will be overriten by the `upload` options, to be sure that the wizard options will be used, create a seperate config file

Reviewer: ```suggestion
> **_NOTE:_** The `wizard` options will be overwritten by the `upload` options, to be sure that the wizard options will be used, create a separate config file.
```
